udata/api/__init__.py

```diff
@@ -278,6 +281,44 @@ class API(Resource):  # Avoid name collision as resource is a core model
     pass
 
 
+oembed_parser = api.parser()
+oembed_parser.add_argument(
+    'url', type=str, help='URL of the resource to embed.', required=True)
+
+
+@api.route('/oembed/', endpoint='oembed')
+class OEmbedAPI(API):
+
+    def get(self):
```

Comment:
`@api.doc(oembed, parser=parser)` to make it appear in the documentation (and whatever is needed to document)